contracts/truefi2/CollateralVault.sol

```diff
@@ -63,19 +65,24 @@ contract CollateralVault is ICollateralVault, UpgradeableClaimable {
     }
 
     function unstake(uint256 amount) external {
-        require(amount <= unstakeableAmount(msg.sender), "CollateralVault: cannot unstake");
+        require(amount <= unstakeableAmount(msg.sender), "CollateralVault: Cannot unstake");
 
         stakedAmount[msg.sender] = stakedAmount[msg.sender].sub(amount);
         stakedToken.safeTransfer(msg.sender, amount);
         emit Unstaked(msg.sender, amount);
     }
 
     function slash(address borrower) external override {
-        // require(msg.sender == liquidator)
-        // require(borrowingMutex.isBanned(borrower))
-        // transfer stakedAmount() to liquidator
-        require(borrower == address(stakedToken)); // silence lint and build warnings
-        revert("Unimplemented!");
+        require(msg.sender == address(liquidator), "CollateralVault: Caller is not the liquidator");
```

Comment:
Should we keep liquidator as a plain address?

Reply:
Can't think of a scenario where this type would carry any value besides being self-documentation. But maybe it's a good enough reason to keep it to be fair